sdk/support/arm-support/package.json

```diff
@@ -29,13 +29,7 @@
   "types": "./types/arm-support.d.ts",
```

Comment:
forget the rollup.config.js ?

Reply:
It seems that it was deleted when merging main in release pr